CMakeLists.txt

```diff
@@ -56,7 +71,7 @@ set(OPENDDS_TARGETS
   OpenDDS::Rtps_Udp
 )
 
-target_include_directories(OpenDDW INTERFACE src)
+#target_include_directories(OpenDDW INTERFACE src)
```

Comment:
Probably need a `BUILD_INTERFACE`/`INSTALL_INTERFACE` separation here. See [`target_include_directories`](https://cmake.org/cmake/help/latest/command/target_include_directories.html), above the "Creating Relocatable Packages" section.

Reply:
Should be good now.